modules/core/test/test_arithm.cpp

```diff
@@ -2620,5 +2620,38 @@ TEST(Core_Magnitude, regression_19506)
     }
 }
 
+using namespace ocl;
```

Comment:
Is it necessary? used?

Reply:
I think `EXPECT_MAT_NEAR` did not work without it. I'll check.

Reply:
I've checked and it is `EXPEC_MAT_NEAR` which requires `ocl` namespace.

Reply:
Could we use related calls with full namespace in this macro?

Reply:
Done! Also rebased to 3.4.